blazor/treeview/check-box.md

```diff
@@ -7,17 +7,19 @@ control: TreeView
 documentation: ug
 ---
 
-# CheckBox in Blazor TreeView Component
+# Check nodes through data binding
 
-The Blazor TreeView component allows to check more than one node in TreeView without affecting the UI's appearance by enabling the `ShowCheckBox` property. When this property is enabled, checkbox appears before each TreeView node text.
+The Blazor TreeView component allows to check more than one node in TreeView without affecting the UI's appearance by enabling the [ShowCheckBox](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfTreeView-1.html#Syncfusion_Blazor_Navigations_SfTreeView_1_ShowCheckBox) property. When this property is enabled, checkbox appears before each TreeView node text.
 
 * If one of the child nodes is not in a checked state, then the parent node will be in an intermediate state.
 
 * If all the child nodes are in checked state, then the parent node's state will also be checked.
 
 * If a parent node is checked, then all the child nodes' state will also be checked.
 
-By default, the checkbox state of parent and child nodes are dependent on each other. For independent checked state, achieve it using the `AutoCheck` property.
+By default, the checkbox state of parent and child nodes are dependent on each other. For independent checked state, achieve it using the [AutoCheck](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfTreeView-1.html#Syncfusion_Blazor_Navigations_SfTreeView_1_AutoCheck) property.
+
+We can also check a specific node by setting the **IsChecked** field to true for the corresponding node in the data source, which specifies the field for the checked state of the TreeView node.
```

Comment:
Have no separate section using IsChecked field? If no, we need to add.

Reply:
Check nodes through data binding - the purpose of the topic is to check nodes using the IsChecked field, sir. So, we no need to add a separate one.